static/src/js/components/KeywordForm/KeywordForm.jsx

```diff
@@ -43,10 +56,30 @@ const KeywordForm = ({
     onFormDataChange(newFormData)
   }
 
-  // Handlesubmit to be completed in MMT-4003
-  // const handleSubmit = ({ submitFormData }) => {
-  //   console.log('Form submitted:', submitFormData)
-  // }
+  const handleSubmit = () => {
```

Comment:
Could we just remove this function and instead just call `setShowModal(true)` directly for the onSubmit then we can rename the `handleFinalSubmit` to maybe handleSave or something

Reply:
No, you cannot directly call setShowModal(true) in the onSubmit prop like this:

onSubmit={setShowModal(true)}
This would cause the setShowModal function to be called immediately when the component renders, not when the form is submitted. Additionally, it would set the onSubmit prop to the result of calling setShowModal(true), which is undefined, rather than to a function that should be called when the form is submitted.